thenewboston_node/accounts/tests/test_transactions.py

```diff
@@ -0,0 +1,31 @@
+from thenewboston_node.business_logic.models import Block
+from thenewboston_node.business_logic.node import get_node_signing_key
+from thenewboston_node.business_logic.tests.base import force_blockchain
+
+API_V1_LIST_TRANSACTIONS_URL_PATTERN = '/api/v1/accounts/{id}/transactions/'
+
+
+def test_can_list_transactions(
```

Comment:
let's have more than 1 transactions in this test. See if you can use `thenewboston_node.business_logic.tests.factories.add_blocks_to_blockchain` for this